### DISCUSSION TITLE
submit a fork

### DISCUSSION THREAD

**Discussion by audioscavenger** [Comfy-Org/ComfyUI-Manager#643] [Q&A]
hi how do you handle forks, for nodes not maintained anymore? I am taking over [save-image-extended-comfyui](https://github.com/audioscavenger/save-image-extended-comfyui/) since the owner is gone for 6 months now.
* owner does not provide any way to contact them, no email nothing
* It has lots of issues unanswered.
* I will try and answer them, but before that, I want to submit my updated and bugfixed version.
* mine is retro-compatible

Can I just submit `custom-node-list.json` and replace his repo/name/url/desc ? How do you know I am legit? do you need the owner to inform you that they agree?

**Comment by ltdrdata** [comment]
> hi how do you handle forks, for nodes not maintained anymore? I am taking over [save-image-extended-comfyui](https://github.com/audioscavenger/save-image-extended-comfyui/) since the owner is gone for 6 months now.
> 
> * owner does not provide any way to contact them, no email nothing
> * It has lots of issues unanswered.
> * I will try and answer them, but before that, I want to submit my updated and bugfixed version.
> * mine is retro-compatible
> 
> Can I just submit `custom-node-list.json` and replace his repo/name/url/desc ? How do you know I am legit? do you need the owner to inform you that they agree?

In that case, I move the existing node to the legacy channel and register the new node in the default channel.
